src/lib/utils/arweave.ts

```diff
@@ -0,0 +1,50 @@
+import Arweave from 'arweave';
+import { decrypt } from 'ecies-geth';
+import { utils } from 'ethers';
+
+export const initArweave = () => {
+  return Arweave.init({
+    host: 'arweave.net', // Hostname or IP address for a Arweave host
+    port: 443, // Port
+    protocol: 'https', // 'https', // Network protocol http or https
+    timeout: 20000, // Network request timeouts in milliseconds
+    logging: false, // Enable network request logging
```

Comment:
Do we want to support arLocal here as well? Maybe these settings should be in the `networkConfig` files?

Reply:
Good call. `networkConfig` would be a good place for this. Should we include that in another ticket?

Reply:
@knoll3 -- It should be a quick update that can be handled in this PR (since this change currently would break arLocal for the local workflow)

Reply:
The arweave config has been restored to point to local host.